keyboards/yandrstudio/r65ble/keymaps/default/rules.mk

```diff
@@ -0,0 +1,11 @@
+# OLED_DRIVER_ENABLE = yes
+VIA_ENABLE = yes
```

Comment:
VIA should not be enabled in the default keymap, and should be in a separate keymap for via.